README.md

```diff
@@ -1,16 +1,16 @@
 # All About ReactJS
 
-### If you want to contribute to this project, consider reading the [contribution.md](https://github.com/DemonDaddy22/all-about-reactJS/blob/master/contribution.md) first to get a better idea on how to contribute to this repo.
+### If you want to contribute to this project, consider reading the [contribution.md](https://github.com/DemonDaddy22/all-about-reactJS/blob/master/contribution.md) first to get a better idea of how to contribute to this repo.
```

Comment:
Did you use any tool for this change, because I think 'on' makes more sense than 'of'?

Reply:
yeah, I checked it on Grammarly too

Reply:
Okay, cool then.